### DISCUSSION TITLE
How to create floating menu on selection?

### DISCUSSION THREAD

**Discussion by deep-c** [remirror/remirror#895] [Q&A]
Hi,

So I'd like to create a popout menu bar that shows above selection of text etc. I presume I could use `extension-positioner` package but the docs are bit light so im not sure how this could be accomplished. A quick example showing the general idea would be welcome.

Awesome project btw, it will be even better I feel when the docs are fleshed out with a bit more examples and information!

Thanks

**Comment by umerjaved178** [comment]
@deep-c any update on it?

**Reply by Amethystafyy282** [reply]
> @deep-c any update on it?

Still looking forward, there was #453.
